src/compas_rhino/__init__.py

```diff
@@ -88,7 +88,7 @@ def _get_ironpython_lib_path_mac(version):
     lib_paths = {
         '5.0': ['/', 'Applications', 'Rhinoceros.app', 'Contents'],
         '6.0': ['/', 'Applications', 'Rhinoceros.app', 'Contents', 'Frameworks', 'RhCore.framework', 'Versions', 'A'],
-        '7.0': ['/', 'Applications', 'RhinoWIP.app', 'Contents', 'Frameworks', 'RhCore.framework', 'Versions', 'A']
+        '7.0': ['/', 'Applications', 'Rhino 7.app', 'Contents', 'Frameworks', 'RhCore.framework', 'Versions', 'A']
```

Comment:
I have a mac to test with, but not Rhino7.  I was wondering if we should keep the reference to the WIP:
```suggestion
        '7.0': ['/', 'Applications', 'Rhino 7.app', 'Contents', 'Frameworks', 'RhCore.framework', 'Versions', 'A'],
        'WIP': ['/', 'Applications', 'RhinoWIP.app', 'Contents', 'Frameworks', 'RhCore.framework', 'Versions', 'A'],
        
```

Reply:
good point, since WIP (ie 8.0) will also exist 👍

Reply:
I think there's no RhinoWIP for 8.0 at the moment according to this [post](https://discourse.mcneel.com/t/thank-you/112993). @beverlylytle please correct me if I didn't find the right download link.  
@gonzalocasas Maybe we can first correct the 7.0 and add the WIP later after testing it. I am using Rhino 7 now, and this change works on my computer. :)

Reply:
No, I don't think there is a Rhino 8 WIP yet, but I was wondering about people who previously downloaded the Rhino 7 WIP.  Is that still usable?  Or did it expire with the release of Rhino 7?

Reply:
oh for my understanding is that once the released version is out, the WIP no longer works. (I remember I read this somewhere before but can't find it now, because people need to buy the licence) :stuck_out_tongue_closed_eyes:

Reply:
oh ok, then ignore my suggestion.

Reply:
the app name is now `Rhino 7`? or did you rename this?

Reply:
@brgcode the app is called Rhino 7 when I downloaded it. I didn't change the name.